### ISSUE TITLE
conditioning the image generation

### ISSUE THREAD

**Issue by sambatra** [odegeasslbc/FastGAN-pytorch#48]
Hi,

This is a great and neat implementation. I was wondering of there is a simple way to implement conditioning (on image label) like a conditional GAN. I did try the standard approach used in cGAN but it was not successful. Could you please point me in the right direction?

Cheers,

**Comment by szy96** [comment]
> 

have you ever solve the problem? I'm also confused about it. Thanks

**Comment by dangtr0408** [comment]
I recently implemented one by conditioning the SLE layers. There are many other approaches as well, like ACGAN, which works great for fewer classes. I recommend looking at Projected GAN, they implemented conditional BatchNorm in their FastGAN.
